std_msgs/Header header
laser_line_extraction/LineSegment[] line_segments
